src/cdk-experimental/selection/row-selection.ts

```diff
@@ -31,7 +31,7 @@ export class CdkRowSelection<T> {
   @Input('cdkRowSelectionIndex')
   get index(): number|undefined { return this._index; }
   set index(index: number|undefined) { this._index = coerceNumberProperty(index); }
-  private _index?: number;
+  protected _index?: number;
```

Comment:
I don't follow why this and some of the ones below have to be protected now.

Reply:
That's because  of this `Types have separate declarations of a private property '_templateLabel'.ts(2415)` though with input renaming via metadata this shouldn't be needed.